lib/katalyst/basic/auth/rails.rb

```diff
@@ -4,9 +4,18 @@ module Katalyst
   module Basic
     module Auth
       class Railtie < Rails::Railtie
+
+        def middleware_enabled?
```

Comment:
how about

```
enable = ENV.fetch("KATALYST_BASIC_AUTH_ENABLED", Rails.env.staging? || Rails.env.uat?)
[true, "yes", "true"].contains?(enable)
```

Reply:
If you move this code to `auth.rb` then you can call it from the rake task too (e.g. `Katalyst::Basic::Auth::Config.enabled?`)

Reply:
reworked method, added your code in new Config class